Sources/GoodNetworking/Session/Logger/LoggingEventMonitor.swift

```diff
@@ -133,3 +138,28 @@ private extension LoggingEventMonitor {
     }
 
 }
+
+public extension LoggingEventMonitor {
+    
+    static private(set) var responseTypeWhiteList: [String] = [
```

Comment:
```suggestion
    private(set) static var responseTypeWhiteList: [String] = [
```

Reply:
dokonca naopak to je aj spravne podla [syntax guidelines od SwiftLintu](https://github.com/masters3d/SwiftLint/blob/5d4a7d7cc182c613485452841112c9ccc970bedc/Source/SwiftLintFramework/Extensions/SwiftDeclarationAttributeKind%2BSwiftlint.swift) 😄 je to napisane trochu komplikovane ale da sa to precitat z toho sourcu